Makefile

```diff
@@ -62,45 +69,65 @@ tls-cert:
                -addext "subjectAltName=DNS:example.com,DNS:www.example.net,IP:10.0.0.1"
 	@echo "Success! find your cert files in the tls/ folder"
 
-generate:
+# delete the tls directory and certs
+.PHONY: tls-delete
+tls-delete:
+	@echo "removing generated tls certs"
+	@rm -rf tls/
+
+# generate a go client from the openAPI spec
+.PHONY: apiclient-gen
```

Comment:
I think this make target, apiclient-delete and the ./scripts/generate.sh can be deleted. I'm pretty sure we don't use it anywhere and is made redundant by buf-gen, etc. Might be good to take the opportunity to remove it.

Reply:
ok, will do